### PR TITLE
Add stress_revision support

### PR DESCRIPTION
**Pull request by aboudreault**
@mambocab and @EnigmaCurry review

### CODE REVIEW

**Comment by aboudreault** [comment]
No rush here. Was just wondering if you got the notification when written in the description. //cc @mambocab @EnigmaCurry

File: tool/cstar_perf/tool/stress_compare.py [hunk 1]
```diff
@@ -146,7 +154,7 @@ def stress_compare(revisions,
                 # Run stress:
                 # (stress takes the stats as a parameter, and adds
                 #  more as it runs):
-                stats = stress(cmd, revision, stats, stress_path=revision_config.get('stress_path', None))
+                stats = stress(cmd, revision, stats, stress_revision=revision_config.get('stress_revision', None))
```
Comment (mambocab) [inline tool/cstar_perf/tool/stress_compare.py]: I don't see `stress_revision` in the parameter list for `stress`.

Comment (aboudreault) [inline tool/cstar_perf/tool/stress_compare.py]: oh right. checking to add it

**Comment by mambocab** [comment]
I can't think of any reason to use `stress_path` now that `stress_revision` exists, but I want to make sure we all agree on that.

I don't have an environment to test this in right now, but this looks correct. If you've run it locally, I'm +1 as long as we're all ok with `stress_path` going away.

**Comment by aboudreault** [comment]
You are correct for stress_path. Ryan and I already agreed to remove it with the addition of stress_revision. So we are all +1 on this.

File: tool/cstar_perf/tool/stress_compare.py (hunk 1 above)
Comment (aboudreault) [inline tool/cstar_perf/tool/stress_compare.py]: For the record, stress_revision is already present.

**Pull request merged**